locales/ko_KR/LC_MESSAGES/duckduckgo.po

```diff
@@ -73,15 +73,15 @@ msgctxt "noresults"
 msgid ""
 "%sClick here%s to try again, if you think there should be results for this "
 "search."
-msgstr ""
+msgstr "%여기%를 클릭해서 다시 시도해 보세요."
```

Comment:
This needs `%s` format placeholders - is this correct?

```suggestion
msgstr "%s여기%s를 클릭해서 다시 시도해 보세요."
```

Reply:
_ fixed in https://github.com/duckduckgo/duckduckgo-locales/pull/34/commits/94ba72613d1aa79fc621421aa2185843584f1785. thankies!